api/column_family.hh

```diff
@@ -68,11 +68,10 @@ struct map_reduce_column_families_locally {
     std::function<std::unique_ptr<std::any>(std::unique_ptr<std::any>, std::unique_ptr<std::any>)> reducer;
     future<std::unique_ptr<std::any>> operator()(replica::database& db) const {
         auto res = seastar::make_lw_shared<std::unique_ptr<std::any>>(std::make_unique<std::any>(init));
-        for (auto& i: db.get_column_families()) {
+        auto column_families = db.get_column_families();
+        for (auto& i: column_families) {
             *res = reducer(std::move(*res), mapper(*i.second.get()));
-            // FIXME:
```

Comment:
Note: now we can get an exception because we access a table that was removed. So we need a lock around table addition and something like database::for_each_table(function that returns a future).